src/App.css

```diff
@@ -58,7 +58,7 @@ body {
 .input-container {
   display: flex;
   justify-content: center;
-  column-gap: 10px;
+  column-gap: 11px;
 }
 
 input {
```

Comment:
- [ ] you have don a fantastic job in your project so far, just a little adjustment to make on your mobile design and your are good to go, kindly make your mobile view look like the design given
your design in mobile :point_down: 
![Screenshot from 2023-01-14 13-22-40](https://user-images.githubusercontent.com/66097371/212471483-e9c6deee-a0d1-4126-a468-55abfd02ee2e.png)
design given :point_down: 
![image](https://user-images.githubusercontent.com/66097371/212471506-f6c97c5a-d5c4-445b-8d79-5583dbea84e6.png)